src/main/java/org/apache/maven/doxia/cli/CLIManager.java

```diff
@@ -93,6 +95,10 @@ class CLIManager {
                 .desc("Input file or directory.")
                 .hasArg()
                 .build());
+        OPTIONS.addOption(Option.builder(REMOVE_IN)
+                .longOpt("removeInputAfterConversion")
+                .desc("Whether to remove the input file(s) after successfull conversion")
```

Comment:
successful

Reply:
Fixed in https://github.com/apache/maven-doxia-converter/pull/85/commits/425bf865d82c871189f146be5c64a75d0d5bf93f.